### ISSUE TITLE
Manage RECALLING->KILLRECALL transition in the REST

### ISSUE THREAD

**Issue by mmascher** [dmwm/CRABServer#5608]
Upon user request (crab kill) we need to detect a task in RECALLING state and put it in KILLRECALL.

Should be as simple as adding a branch to this if https://github.com/dmwm/CRABServer/blob/master/src/python/CRABInterface/DataWorkflow.py#L569-L576

**Comment by belforte** [comment]
depends on #5606 
if we close that as "will not do", will close this as well.

**Comment by belforte** [comment]
On hold. Possibly forever. See #5606

**Comment by belforte** [comment]
we can and need to kill recall requests now that we use Rucio. Removing HOLD and connecting to https://github.com/dmwm/CRABServer/issues/7406 and give same labels

**Comment by belforte** [comment]
time to do. We do need this
